job-utilities/job-router/README.md

```diff
@@ -0,0 +1,43 @@
+# job-router
+
+Python application to route scheduled jobs to ECS services on AWS infrastructure
+
+## Building
+
+A docker image for local use can be built using 'docker build --platform linux/amd64 -t image-name:image-tag .'
+Putting together a package for deployment to Lambda is done with the following steps:
+
+```
+mkdir package
+pip install --target ./package -r requirements.txt
+cd package
+zip -r ../deployment_package.zip .
+cd ..
+zip deployment_package.zip lambda_function.py
+```
+
+## Deploying
+
+A manual deployment of the zip package is easy. Simply go to the job-router Lambda and upload the zip package of code to update it.
```

Comment:
Does this mean that there will be no automated deployment?

Reply:
There will be. I'm getting a build plan made and will use the standard tools we have for deploying lambdas. There's no need to go into detail about our internal automated deployment, so I put the manual instructions in. There's probably a aws-cli command that could do it easier, so I'm actually going to look into adding that

Reply:
Or I might write a script that can deploy the lambda code, I guess I need to check if our standard tools are really a good fit in this particular case